elisp/edts/edts-navigate.el

```diff
@@ -106,9 +106,9 @@ directive."
 
 (defun edts-macro-under-point-p ()
   "Return non nil if the form under point is a macro."
-  (save-excursion
-    (beginning-of-thing 'symbol)
-    (equal ?? (char-before (point)))))
+  (let ((bounds (edts-atom-at-point)))
```

Comment:
If you want to use `edts-atom-at-point` should ensure that `case-fold-search` is non-nil, otherwise you won't be able to match eg `?FOO`

Reply:
Thank for the comment,
But this change is similar with  ` edts-record-under-point-p` and macro in erlang should be case-sensitive, isn't it?

Reply:
Yeah, but it's not about whether the macro is case sensitive or not, it's about whether your regexp will be able to recognize a macro or not. `edts-atom-at-point` will not match a symbol starting with an upper character since that is not allowed for atoms

Reply:
I tested 'edts-atom-at-point' and it works well with uppercase macro (like ?FOO).
Maybe because you used (re-search-backward "[^@_a-zA-Z0-9]") in 'edts-atom-at-point'?
Btw, can you tell me why the Travis CI always failed for this commit? Is it because what you mentioned above?

Reply:
@tjarvstrand may I know your view?